tt_metal/impl/dispatch/kernels/cq_dispatch.cpp

```diff
@@ -201,6 +201,77 @@ void process_write() {
     }
 }
 
+template<bool is_dram>
+FORCE_INLINE
+void process_write_paged() {
+    volatile tt_l1_ptr CQDispatchCmd *cmd = (volatile tt_l1_ptr CQDispatchCmd *)cmd_ptr;
+
+    uint32_t page_id = cmd->write_paged.start_page;
+    uint32_t base_addr = cmd->write_paged.base_addr;
+    uint32_t page_size = cmd->write_paged.page_size;
+    uint32_t pages = cmd->write_paged.pages;
+    uint32_t data_ptr = cmd_ptr + sizeof(CQDispatchCmd);
+    uint32_t write_length = pages * page_size;
+    InterleavedAddrGen<is_dram> addr_gen;
```

Comment:
Nit: Would be much better to have this be templated by InterleavedBufferType enum/enum class than just bool (out of scope of this PR, just wanted to track).

Reply:
Ah, I get the feeling these low level FD2.0 commands are purposely trying to be unaware of those types.. the spec has a (effective) "bool" afterall which is straight plugged into the template arg:
```
struct CQDispatchWritePagedCmd {
    uint8_t pad1;
    uint8_t is_dram;          // one flag, false=l1
```